cookbooks/bcpc-hadoop/metadata.rb

```diff
@@ -7,9 +7,7 @@
 version          '0.1.0'
 
 depends 'bcpc', '= 0.1.0'
-depends 'bach_krb5', '= 0.1.0'
-depends 'smoke-tests', '= 0.1.0'
-depends 'bach_opentsdb', '= 0.1.0'
```

Comment:
i thought that `bcpc-hadoop` was like one of the ultimate wrapper cookbooks.  `bcpc-hadoop` should depend on `bach_opentsdb`. not the otherway around.

Reply:
I'm not sure there's ever been a clear pattern of what depends on what.  They all depend on the same blob of mutable state, so it's hard to say.

Reply:
then we should move out all the wrapper cookbooks out of bcpc-hadoop then

moving to roles just makes the dependency implicit.  which are then harder to debug.

Reply:
Not only should we not be adding new implicit dependencies to roles, In a perfect world we would using roles entirely.